packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -148,6 +150,15 @@ export default class VTable {
         }
     }
 
+    private recalculateCellHeight(td: HTMLTableCellElement) {
+        if (this.isAEmptyCell(td) && td.rowSpan && td.rowSpan > 1) {
+            for (let i = 1; i < td.rowSpan; i++) {
+                const br = document.createElement('br');
+                td.appendChild(br);

```

Comment:
why do this?

Reply:
hmm, ok. But I feel maybe the better way is to set the height of cell, but not use BR to fill it.

We can go with this for now.